pmoves/services/gateway-agent/app.py

```
@@ -0,0 +1,527 @@
+    SECRETS_MAP = {
+        "HOSTINGER_API_KEY": "hostinger",
+        "TAILSCALE_AUTHKEY": "tailscale",
+        "TAILSCALE_API_KEY": "tailscale",
+        "N8N_API_KEY": "n8n",
+        "POSTMAN_API_KEY": "postman",
+        "E2B_API_KEY": "e2b",
+        "VENICE_API_KEY": "cipher",
+        "OPENAI_API_KEY": "openai",
+        "ANTHROPIC_API_KEY": "anthropic",
+        "GEMINI_API_KEY": "gemini",
+    }
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)